tools/intersmash-tools-provisioners/pom.xml

```diff
@@ -188,6 +188,7 @@
                 <configuration>
                     <target>target/generated-sources</target>
                     <source>src/main/resources/crds</source>
+                    <enumUppercase>false</enumUppercase>
```

Comment:
Actually it is needed because of the Infinispan CRD design, which exposes similar values for one (or more?) enums, e.g.: see https://github.com/infinispan/infinispan-operator/blob/2.3.4.Final/config/crd/bases/infinispan.org_infinispans.yaml#L1473-L1477
But this is a good point, I'll add a comment there.

Reply:
Done!